mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule OriginSimulator.MixProject do
       {:plug_cowboy, "~> 2.0"},
       {:poison, "~> 4.0"},
       {:httpoison, "~> 1.5"},
-      {:distillery, "~> 2.0", runtime: false},
+      {:distillery, "~> 2.0"},
```

Comment:
Are we updating these dependencies in another PR? For example Cowboy here is 2.4 yet there is 2.8 from 2020.

Reply:
I will do these in a seperate PR.

Reply:
It turns out, I have actually done all the leg work for these in the PRs I just need to do some touch ups. 
With regards to your example which is the case for most of the deps, with `{:cowboy, "~> 2.4"}` this will allow any version of cowboy, above 2.4 and below 3.0 as you can see: (Found in iex with `h Version`)

```
~>           | Translation             
~> 2.0.0     | >= 2.0.0 and < 2.1.0    
~> 2.1.2     | >= 2.1.2 and < 2.2.0    
~> 2.1.3-dev | >= 2.1.3-dev and < 2.2.0
~> 2.0       | >= 2.0.0 and < 3.0.0    
~> 2.1       | >= 2.1.0 and < 3.0.0  
```

so 2.8 is being used and you can see in the mix.lock file but I am going to go through and update the versions in the mix.exs files